README.md

```diff
@@ -42,6 +42,45 @@ Explore our [collaborative examples](https://github.com/drifting-in-space/y-swee
 
 All examples are open source and live in this repository, within [/examples](https://github.com/drifting-in-space/y-sweet/tree/main/examples).
 
+## Usage
+
+Check the [vanilla js example](/examples/vanilla/) for more details.
+
+### On the server
+``` js
+import { createDoc } from '@y-sweet/sdk'
+// Get the client token from the y-sweet server. This is basically the user's "password" to edit the "myDoc123" doc.
+app.get('/client-token', async (req, res) => {
```

Comment:
I think it might be simpler / clearer to drop the `app.get()` part and just have something like:

```ts
import { createDoc } from '@y-sweet/sdk'

// Get the client token from the y-sweet server. This is basically the user's "password" to edit the "myDoc123" doc.
const clientToken = await createDoc('myDoc123', CONNECTION_STRING)
```